services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
```

Comment:
Error handling is not properly implemented. Instead of simply re-throwing the error, consider logging the error and providing a more user-friendly error message.